gxa/src/main/java/uk/ac/ebi/atlas/controllers/rest/api/AssayDesignController.java

```diff
@@ -1,34 +0,0 @@
-package uk.ac.ebi.atlas.controllers.rest.api;
-
-import com.google.common.base.Joiner;
-import org.springframework.context.annotation.Scope;
-import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.ResponseBody;
-import uk.ac.ebi.atlas.model.arraydesign.ArrayDesignDao;
-
-import javax.inject.Inject;
-import java.util.Map;
-
-@Controller
-@Scope("request")
-public class AssayDesignController {
```

Comment:
What was this used for?

Reply:
Maria used it for something on her pipeline. Sorry but I can’t remember the specifics.